README.md

```diff
@@ -126,30 +126,17 @@ The **API endpoint** can be used to:
   scenarios; applies a cgroup/namespace isolation barrier and then
   drops privileges.
 
-## Supported platforms
+## Tested platforms
```

Comment:
I lean against including any requirement in documentation that we can't or don't test. Do you know the reasoning behind why the minimum requirement would be Skylake or Neoverse N1?

Reply:
Because those are the oldest CPUs we know about, but it may work on older ones..

Reply:
I think to a reader this may seem an arbitrary line though. If these are the oldest CPUs we have had Firecracker working on at some point, we do not still know if it works on these CPUs.

Although I don't know if its useful to a reader, I think it we could add a line like:

> The oldest CPU architectures that have been tested running a version of Firecracker are Skylake and Neoverse N1.